docs/testing.md

```
@@ -1,5 +1,23 @@
 # Testing
 
+<!-- vim-markdown-toc GFM -->
+
+* [Running tests](#running-tests)
+    * [Run all tests](#run-all-tests)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)